examples/nidmm/continuous-acquire-graph-multiple-points.py

```diff
@@ -0,0 +1,198 @@
+# 
+# This example performs continuous multipoint acquisition and prints the number of points read and average value of measurements.
+#
+# The gRPC API is built from the C API.  NI-DMM documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niDMM\Documentation\English\DMM.chm
+
+#
+# Getting Started:
+#
+# To run this example, install "NI-DMM Driver" on the server machine.
+# Link : https://www.ni.com/en-us/support/downloads/drivers/download.ni-dmm.html
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# Refer to the NI-DMM gRPC Wiki to determine the valid channel and resource names for your NI-DMM module.
+# Link : https://github.com/ni/grpc-device/wiki/NI-DMM-C-Function-Reference
```

Comment:
I don't think we'll have a DMM C-function reference wiki at the end of Q2. All that should be moving to ni.com. Work with Kevin Reimnitz to work out the details.

Reply:
removed that link, I will add the link to reference in later PR

Reply:
I added the link to the ni.com page